Cannot find module 'systeminformation'

Hi,

i'm interested in your extension, but unfortunately it doesn't run on my machine.

From what appears to me, you need to ship the extension with the required dependencies, in case `systeminformation`.

The vscode console has this warning:
> Activating extension 'mutantdino.resourcemonitor' failed: Cannot find module 'systeminformation'.

From Microsoft documentation i've found: [Using Node.js Modules with Extensions](https://code.visualstudio.com/docs/extensionAPI/patterns-and-principles#_using-nodejs-modules-with-extensions)

--- a/package.json
+++ b/package.json
@@ -48,7 +48,10 @@
             }
         }
     },
-    "repository": ,
+    "repository": {
+        "type" : "git",
+        "url" : "https://github.com/Njanderson/resmon.git"
+    },
     "scripts": {
         "vscode:prepublish": "npm run compile",
         "compile": "tsc -p ./",

--- a/package.json
+++ b/package.json
@@ -59,6 +59,9 @@
         "postinstall": "node ./node_modules/vscode/bin/install",
         "test": "npm run compile && node ./node_modules/vscode/bin/test"
     },
+    "dependencies": {
+        "systeminformation": "^3.0.0"
+    },
     "devDependencies": {
         "typescript": "^2.6.1",
         "vscode": "^1.1.6",

